vignettes/list_of_functions.Rmd

```diff
@@ -40,10 +40,11 @@ library(effectsize)
 library(modelbased)
 library(see)
 library(report)
+library(easystats)
 
 all_funs <- c()
 
-for (package in easystats:::.packages_on_cran()) {
+for (package in c(easystats:::.packages_on_cran(), "easystats")) {
```

Comment:
why not add "easystats" in `.packages_on_cran()`?

Reply:
Done.

That's what I had done in my initial commit, but managed to muck it up somehow in the later commits.